tuw_nlp/grammar/text_to_4lang.py

```diff
@@ -18,11 +18,14 @@
 class TextTo4lang():
     def __init__(self, lang, nlp_cache, cache_dir=None):
         if lang == 'de':
-            nlp = CustomStanzaPipeline(
+            nlp = stanza.Pipeline(
```

Comment:
The CustomStanzaPipeline included a processor for fixing some sentence segmentation errors of the German model, are you removing that on purpose?

Reply:
It was a mistake, thank you for noticing :) I mixed up CustomStanzaPipeline with CachedStanzaPipeline...